muda/deformers/clipping.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+# CREATED:2020-06-17 by Jatin Khilnani <Jatin.Khilnani@nyu.edu>
+"""Clipping (waveform/loudness distortion) transformations"""
+
+import numpy as np
+
+from ..base import BaseTransformer, _get_rng
+
+__all__ = ["Clipping", "LinearClipping", "RandomClipping"]
+
+
+class AbstractClipping(BaseTransformer):
+    """Abstract base class for clipping
+
+    This contains the deformation function 
+    but does not manage state or parameters.
+    """
+
+    def __init__(self):
+        BaseTransformer.__init__(self)
+
+        # Build the annotation mappers
+#         self._register(".*", self.deform_times)
+#         self._register("tempo", self.deform_tempo)
```

Comment:
no annotation mappers needed for this one

Reply:
Missed removing this commented out portion, would take care of it.

Reply:
Removed.